matminer/featurizers/site.py

```diff
@@ -622,3 +623,55 @@ def citations(self):
 
     def implementors(self):
         return ['Qi Wang']
+
+
+class EwaldSiteEnergy:
+    """Compute site energy from Coulombic interactions
+
+    User notes:
+        - This class uses that `charges that are already-defined for the structure`.
+          Consider assigning them using the ``add_charges_*`` from ``pymatgen``
```

Comment:
I'd remove the "Consider assigning them..." comment - not sure the user will figure out what to do. We should just add some utils to matminer for assigning charge and having the user use that.